packages/noya-state/src/reducers/interaction.ts

```diff
@@ -52,7 +52,9 @@ export type InteractionAction =
   | [type: 'updateMoving', point: Point]
   | [type: 'updateScaling', point: Point]
   | [type: 'updatePanning', point: Point]
-  | [type: 'enablePanMode'];
+  | [type: 'enablePanMode']
+  | [type: 'modifyColor']
```

Comment:
Remove these since they're unused

Reply:
I can confirm that it wasn't used. I deleted it.